package.json

```diff
@@ -178,7 +178,7 @@
   },
   "dependencies": {
     "@brave/brave-ui": "0.40.6",
-    "@brave/leo": "github:brave/leo#5e79418b1937e4c524f6d40c58e14f34be5711a8",
+    "@brave/leo": "github:brave/leo#a892ee311bcd584739ca473aa976974383020a0a",
```

Comment:
Any chance we can bump Nala separately, so we can generate a changelog PR?

Reply:
yes, this is temporary (see commits)